src/pages/_docs/modals.md

```diff
@@ -7,9 +7,11 @@ menu: docs.components.modals
 ## Default markup
 
 {% capture code %}
-<div class="modal-dialog" role="document">
-    <div class="modal-content">
-        {% include parts/modals/simple.html %}
+<div class="modal{% hide %} d-block position-relative{% endhide %}" tabindex="-1">
```

Comment:
never seen `hide`, what is it? same as `comment`?

Reply:
It is a custom tag for the documentation generation system to hide text or code in the example/demo code section whilst still using it to render the example itself. see: https://github.com/tabler/tabler/blob/main/src/pages/_includes/example.html#L2

Those two classes are only required to render the modals on the docs page as `.modal` has a `display: hidden` and `position: fixed`, which is expected behaviour as modals are hidden by default. But as they should not be added by end users we hide them in the demo code.

Reply:
interesting approach with the custom tag ... no wonder I couldn't find it in the liquid documentation.
thanks for the link, haven't seen that part yet 👍 

and thanks for fixing the modal, very nice!